src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/RuleValidation/ValidationRulesAccess.swift

```diff
@@ -171,3 +180,16 @@ public struct ValidationRulesAccess: ValidationRulesAccessing {
         return .success(firstPassedRule)
     }
 }
+
+public extension CertLogicEngine {
+    public convenience init (rules: [Rule]) {
+        guard let schemaURL = Bundle.module.url(forResource: "dcc-validation-rule", withExtension: "json"),
+              let schemaData = FileManager.default.contents(atPath: schemaURL.path),
+              let schemaString = String(data: schemaData, encoding: .utf8) else {
+            // Log.error("Failed to decode the JSON file", log: .localData, error: nil)
+            self.init(schema: "", rules: rules)
```

Comment:
I don't think that we should just go ahead and pass an empty schema to `CertLogicEngine`.
I the other cases where we initialized `CertLogicEngine` and the access to the schema fails, we have a clean error handling. Please have a look at `ValidationRulesAccess` line 61.

Reply:
I think you can remove this initializer completely. The `CertLogicEngine` is initlizied in the implementation of `ValidationRulesAccess`